### PR TITLE
VerifyScreen

### PR DESCRIPTION
**Pull request by aahenry51**

### CODE REVIEW

**Comment by aahenry51** [comment]
image

**Comment by aahenry51** [comment]
image

**Comment by aahenry51** [comment]
image

**Comment by Dulaya** [comment]
- Very important: Remove the nested Container. You only need one Container on the outside.
- You have a few type errors.
- Add background to container.
- Okay to remove `TVerifyScreenProps` fromt Verify.type.ts if you're not using it:

      export type TVerifyScreenProps = StackScreenProps<
        TAuthNavigationRoutes,
        EAuthNavigationRoutes.VERIFY
      >;

- Your Button should have rounder borderRadius.
- Make sure you use camelCase:
   - PinContainer --> pinContainer
- Add fontFamily and size to subtext, subtext2 and subtext2button.
- Change color.red to color.primary in subtext2button.
- Why is red the same as primary in color.ts? 
- Optional: 
   - Consider changing the name subtext2button to something like subText2Link because it's not a button.
   - Consider adding shading/shadow to the number inputs.
- Verification works very smooth. Good job. ✔️

**Comment by aahenry51** [comment]
> 

1.Done
2.Done
3.Done
4.Done
5. I set the button radius to lg
6.Done
7.Done
8. I'm not sure but I can change the color to a red one in color.ts
9. I changed the name and added an on press event

**Comment by aahenry51** [comment]
I'm getting this bottom white space 
image

File: mobile/src/screens/verify/Verify.view.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import {size} from '../../theme';
+
+export const VerifyScreenView: FC<TVerifyScreenViewProps> = ({
+  title = 'Verification Code',
+  emailName = 'test@gmail.com',
+  pinEntry,
+  OnPressResend,
+}) => {
+  return (
+    <Container>
```
Comment (lasharela) [inline mobile/src/screens/verify/Verify.view.tsx:15]: header will be as prop

**Review by lasharela — CHANGES_REQUESTED** [review]

**Review by Dulaya — APPROVED** [review]
Thanks for fixing.

**Review by raynesmike — APPROVED** [review]

**Review by raynesmike — APPROVED** [review]

**Review by Josetf2396 — APPROVED** [review]

**Pull request merged**